src/encoder.rs

```diff
@@ -535,6 +538,8 @@ pub struct FrameInvariants<T: Pixel> {
   pub h_in_imp_b: usize,
   /// Intra prediction cost estimations for each importance block.
   pub lookahead_intra_costs: Box<[u32]>,
+  /// Inter prediction cost estimations for each importance block.
```

Comment:
This is confusing because it's computing inter-costs without taking the motion vectors into account (which is wrong?), and because block importance now uses different inter-costs (with motion vectors). Is there a mistake here?

Reply:
Good find, this comment is wrong, and more importantly failing to check the MVs is probably wrong as well. That may be behind some of the false positives showing in objective-1-fast.